test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -214,24 +227,40 @@ func testPass(testConfig *testConfig, props map[string]string) error {
 	}
 
 	for _, testsuite := range testsuites {
+		buildDir, err := getEBPFBuildDir()
+		if err != nil {
+			return fmt.Errorf("getEBPFBuildDir: %w", err)
+		}
+		bpfDir := filepath.Join(testConfig.testDirRoot, buildDir)
+
+		var testContainer *testContainer
+		if testConfig.inContainerImage != "" {
+			testContainer = newTestContainer(testConfig.inContainerImage, testsuite, bpfDir)
+			if err := testContainer.start(); err != nil {
+				return fmt.Errorf("error creating test container: %w", err)
+			}
+		}
```

Comment:
Since this options applies to all the testsuites, can we do this once outside the loop?

Reply:
I could do this, for our case this doesn't change anything since we have only one testsuite but yeah let me fix that